keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/credential/Ssha1PasswordHashProvider.java

```diff
@@ -0,0 +1,74 @@
+package io.cloudtrust.keycloak.credential;
+
+import com.google.common.primitives.Bytes;
+import org.keycloak.common.util.Base64;
+import org.keycloak.credential.hash.PasswordHashProvider;
+import org.keycloak.models.PasswordPolicy;
+import org.keycloak.models.credential.PasswordCredentialModel;
+
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+/**
+ * Implementation of the Salted SHA1 password hash algorithm.
+ */
+public class Ssha1PasswordHashProvider implements PasswordHashProvider {
+    private static final String SSHA1_PREFIX = "{SSHA}";
```

Comment:
{SSHA1} ?

Reply:
I think {SSHA} matches the spec of the Jira.